api/app.py

```diff
@@ -2,13 +2,13 @@
 import yaml
 from flask import Flask, jsonify
 from flasgger import Swagger
-from api.prints.swagger.swag import SwaggerDoc
-from api.db import db
-from api.elastic import elastic
-from api.prints import base, search, uuid
+from prints.swagger.swag import SwaggerDoc
+from db import db
+from elastic import elastic
+from prints import base, search, uuid
 
 def loadConfig():
-    with open('config.yaml', 'r') as yamlFile:
+    with open('config.yaml-dist', 'r') as yamlFile:
```

Comment:
if you want to reduce dependency on working directory, , make the path relative to __file__ 
for example, the absolute path to the directory containing app.py is `os.path.dirname(__file__)`

Reply:
Unsure of what you mean. Is it possible for you to change the code yourself?

Reply:
I looked at this some more, and it's odd. the config method duplicates the same method in main() I assume you launch the service in main, right? Often a program will take a path to the config file in a parameter. this program assumes the config file is in the working directory, which is fine as it is. But probably not a good idea to be loading it in two places.

Reply:
So there's a main.py and an app.py. They do two different things and you would call either or. Both won't run at the same time. Main.py is called for populating the database, app.py is for launching the API.

Reply:
https://en.wikipedia.org/wiki/Code_refactoring

Reply:
Okay so the repo does provide the config file and the readme tells users to fill it out with the relevant information prior to running the program. Because the config file is included with the source code in the first place for the user to modify, I believe that's why the loadConfig functions in main.py and app.py just open the provided config file directly. But you're saying it's better if we change it so that the user can provide the path to a config file instead?